ddl/ddl_api.go

```diff
@@ -2432,6 +2441,14 @@ func (d *ddl) AlterTable(ctx context.Context, sctx sessionctx.Context, ident ast
 		if !sctx.GetSessionVars().EnableChangeMultiSchema {
 			return errRunMultiSchemaChanges
 		}
+
+		if isDropIndexes(validSpecs) {
+			if err = d.DropIndexes(sctx, ident, validSpecs); err != nil {
```

Comment:
I think we'd better put it to line 2457, and add a "ActionDropIndexes" case.

Reply:
I modified the `isSameTypeMultiSpecs` function and add some test cases for drop both primary key and index.